tests/export/html/test_markup_compatibility.py

```diff
@@ -91,3 +91,71 @@ def test_textbox_with_content_outside_of_textbox(self):
             </p>
         '''
         self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_only_text(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <r>
+                                <t>CCC</t>
+                            </r>
+                        </Fallback>
+                    </AlternateContent>
+                    <t>DDD</t>
+                </r>
+                <r><t>EEE</t></r>
+            </p>
+        '''
+
+        document = WordprocessingDocumentFactory()
+        document.add(MainDocumentPart, document_xml)
+
+        expected_html = '''
+            <p>AAABBBCCCDDDEEE</p>
+        '''
+        self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_a_table(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <tbl>
```

Comment:
The easiest way to do this would be to destroy the choice element and un wrap the alternative content and the fallback content. I see two problems with that:

1. AlternateContent would basically need to be a child of all elements in order to work correctly
1. I have no idea how to alter the XML structure. I see that we are doing it for hyperlinks, I have no idea how it works.